### ISSUE TITLE
Output saving figure

### ISSUE THREAD

**Issue by tischi** [NEUBIAS/training-resources#351]
<img width="834" alt="image" src="https://user-images.githubusercontent.com/2157566/158249664-cabf191d-7b9f-49ca-975d-009fc2ecac97.png">

A few comments:

1. Could we just remove the input image? I think this is obvious from the label mask that such an input image must have existed. Then one could arrange the outputs horizontally, which would render nicer on a website, which is relatively more broad.
2. The X, Y and XM, YM columns in the measurements table are identical. Could the XM and YM be replaced by something more interesting such as area or circularity?
3. The ROI Set table: I wonder whether such a ROI table is typical? Anyway, the current table does not contain too interesting information. Maybe we could remove the table? Instead of the table, maybe it would be more important to visually make clear that ROIs are conceptually different from the label mask? Maybe if it clearly were polygons? It would also be interesting to think about how they are actually stored. Maybe, in the case of polygons a list of linked coordinates? Or in the case of rectangles it could be the center points and width and height....

@AnniekStok @manerotoni Do you have any ideas or opinions on this?

**Comment by AnniekStok** [comment]
@tischi, I agree with your comments. I made a new version to fix point 1, and 2. About point 3 I am not sure yet how to visualize this. 
![image](https://user-images.githubusercontent.com/32863964/158326685-bef4448a-5320-495a-a69b-b67065b7524b.png)

For me saving a ROI set is very handy for 2 reasons: 
1. It is much smaller compared to label images. You can just drag and drop the entire ZIP folder. In the current example the label image is 80 KB, the ROIset 3 KB. ROIs are also very handy for making overlays. With label images you'd have to open the data twice (once raw, once as labels, and overlay the two images)
2. I often work with different kinds of ROIs that partially overlap and are not so easily visible as labels, for example indeed also point rois: 
![image](https://user-images.githubusercontent.com/32863964/158327082-f5f66459-1728-4752-8ff5-b9c2ee81613c.png). 
In my case I often want to see the outline instead of 'filled' labels, which would be an additional step to create using label masks. 

So I think we should make clear somehow that it might make sense to consider data size and type of measurements when deciding how to store the results (labels or ROIs).

**Comment by tischi** [comment]
You are making very good points, which I think should go into a ROI module.
I made an issue: https://github.com/NEUBIAS/training-resources/issues/353

For the output saving module we should not go into these discussions, but still have something in the figure and the activities that makes practical sense.

**Comment by tischi** [comment]
Hi @AnniekStok 
Could you please still change the figure to have a better example for the ROIs?
Maybe just a zoomed in Screenshot on you Polyp with the line ROIs overlayed?

**Comment by AnniekStok** [comment]
Ok I have put in the polyp now (#392)
